### DISCUSSION TITLE
Can notifcations be configured to only send details when it has a certain level of  Severity

### DISCUSSION THREAD

**Discussion by whiteninja76** [DependencyTrack/dependency-track#4482] [Q&A]
What i'm trying to do is to just sent a notification if the Severity of new Vulnerability is critical..
Any guidance on how complicate it is to set this up?

**Comment by msymons** [comment]
Try using Policies to achieve this.  It should not be complicated to set up.

Bear in mind that policies respect tags.  One way in which this can be useful is to ensure that a policy that you are testing out can be restricted to (say) a single project until such time as you are happy that the policy does what you want.

**Reply by whiteninja76** [reply]
how do you then send a notification on a particular policy.. 
i'm trying to setup the notification so that if i get a critical vun its gets set to a specific webhook, if its a high or a critical gets send to a different webhook.

**Reply by ottramst** [reply]
I'm also curious if sending notifications on a particular policy violation is possible
